src/pages/register.js

```diff
@@ -0,0 +1,127 @@
+import React, {Component} from 'react';
+import {Text, Button, FormField, TextInput} from "grommet";
+import request from "superagent/lib/client";
+
+const api_endpoint = '';
+
+export default class Register extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      nickname: '',
+      password: '',
+      password_confirmation: '',
+      id: '',
+      errors: {}
+    };
+  }
+
+  onChangeEmail = event => {
+    this.setState({email: event.target.value});
+  };
+
+  onChangeNickname = event => {
+    this.setState({nickname: event.target.value});
+  };
+
+  onChangePassword = event => {
+    this.setState({password: event.target.value});
+  };
+  onChangeConfirmPassword = event => {
+    this.setState({password_confirmation: event.target.value});
+  };
+
+  handle_register_response(res) {
```

Comment:
Same. 
We could refactor this since it's used elsewhere, no?

Reply:
It's such a small function and although it's repeated it's still too specific to refactor in my opinion.